### PR TITLE
feat: enhance waku cli

### PR DESCRIPTION
**Pull request by Himself65**
``` shell
> waku --version  
0.11.1
```

### CODE REVIEW

**Comment by Himself65** [comment]
I don't know why it failed. Could you please give me the log file?

**Comment by dai-shi** [comment]
> I don't know why it failed. Could you please give me the log file?

The config is changed for #53. Once it's merged into main, you can merge it here.

File: src/cli.ts [hunk 1]
```diff
@@ -27,6 +31,11 @@ switch (cmd) {
   case "start":
     runStart();
     break;
+  case "--version": {
```
Comment (dai-shi) [inline src/cli.ts:34]: I feel like we should consider adding some CLI lib. (Same applies to `create-waku`.)
I will look for something that fits well.

**Review by dai-shi — COMMENTED** [review]
I could merge it, as long as it's short-lived until we refactor soon. (because I want to add more options in the future like `--disable-ssr` 😉 )

But, I'm curious if this is super important and what's your use case.

File: src/cli.ts (hunk 1 above)
Comment (Himself65) [reply src/cli.ts:34]: Nodejs has official support for args. If you like it I could add the support

https://nodejs.org/dist/latest-v18.x/docs/api/util.html#utilparseargsconfig

Comment (dai-shi) [reply src/cli.ts:34]: Oh, that looks good. I think we can use it meanwhile.

**Comment by dai-shi** [comment]
Would you try using https://nodejs.org/dist/latest-v18.x/docs/api/util.html#utilparseargsconfig ?

I'm still curious about your use case.

File: src/cli.ts [hunk 2]
```diff
@@ -2,33 +2,67 @@
 
 import fs from "node:fs";
 import path from "node:path";
+import { parseArgs } from 'node:util'
+import { createRequire } from 'node:module'
+import * as process from 'process'
```
Comment (dai-shi) [inline src/cli.ts:7]: Can we make it this for consistency?
```suggestion
import process from 'node:process'
```

Comment (Himself65) [reply src/cli.ts]: My mistake, process is a global var

**Review by dai-shi — APPROVED** [review]
LGTM!

**Pull request merged**